src/platform/Linux/CHIPDevicePlatformEvent.h

```diff
@@ -67,6 +69,11 @@ struct ChipDevicePlatformEvent
 {
     union
     {
+        struct
+        {
+            unsigned int mAdapterId;
```

Comment:
init this value.

Reply:
This value is initialized here: https://github.com/arkq/connectedhomeip/blob/bluez-object-manager-signals/src/platform/Linux/BLEManagerImpl.cpp#L816

It's not possible to inline initialize this structure. It has to be trivial since it's used in the `ChipDeviceEvent`.

Reply:
thanks